### DISCUSSION TITLE
live changes in cp with forge and DO

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#4589] [Q&A] [answered]
My setup is Laravel Forge/Digital Ocean and bitbucket repo, I have quick deploy setup with a deploy script that looks like this:

```
cd /home/forge/mysite.com
git pull origin master
npm install && npm run production
composer install
php please cache:clear
php artisan config:cache
```
So far this works as it should to deploy local changes pushed to my repo. What I'm wondering is what happens if I make changes in the live version CP. I'm assuming those changes don't get automatically pushed to my repo. Do I simply run a git push in the commands section of forge?

The obvious answer is probably try and find out, but I'm still learning git and don't want to screw something up that I can't fix!

**Comment by duncanmcclean** [comment, marked as answer]
If you're using the Git integration built into Statamic, you can configure it to automatically push any changes made by Statamic on the server to your Git repo. Add this to your `.env`.

```
STATAMIC_GIT_AUTOMATIC=true
```

You can also add a few lines to your Forge deploy script so any pushes to the repo don't trigger a full redeploy. Documented over on [Statamic.dev](https://statamic.dev/git-automation#forge).

**Reply by fredbouin** [reply]
oh interesting, i got the git automation setup but the redeployment part was throwing me off. I'll give this a go.
